docker-app/worker_wrapper/wrapper.py

```diff
@@ -392,9 +415,9 @@ def _prepare_deltas(self, deltas: Iterable[Delta]):
 
         client_pks_map = {}
 
-        for delta_with_modified_pk in local_to_remote_pk_deltas:
-            key = f"{delta_with_modified_pk['client_id']}__{delta_with_modified_pk['content__localPk']}"
-            client_pks_map[key] = delta_with_modified_pk["last_modified_pk"]
+        for delta in local_to_remote_pk_deltas:
+            key = f"{delta['client_id']}__{delta['content__localPk']}"
+            client_pks_map[key] = delta["last_modified_pk"]
 
```

Comment:
also only formatter gone wild

Reply:
There was a recent commit by @m-kuhn who changed that. Why the renaming was required?

Reply:
I guess you are branching from outdated master you might get these?

Reply:
yep correct